src/crud.js

```diff
@@ -1,31 +1,78 @@
-import './style.css';
+const listTasks = JSON.parse(localStorage.getItem('tasks')) || [];
+const listCard = document.getElementById('list');
+const insertInput = document.getElementById('insert');
+const enterBtn = document.getElementById('enterBtn');
 
-const listTasks = [
-  {
-    description: 'Going to Office',
-    completed: false,
-    index: 0,
-  },
-  {
-    description: 'Meeting with ali',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'Go to market',
+const updateIndexes = () => {
+  listTasks.forEach((task, index) => {
+    task.index = index;
+  });
+};
+
+const updateLocalStorage = () => {
+  localStorage.setItem('tasks', JSON.stringify(listTasks));
+};
+
+const deleteTask = (index) => {
+  listTasks.splice(index, 1);
+  updateIndexes();
+  updateLocalStorage();
+};
```

Comment:
- [ ] As you may notice, when deleting a task, there is this weird behavior in your implementation which is that a task is not been deleted, but when we try to add, that it the time a task is being deleted, Kindly make sure you fix that.